e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -1707,6 +1708,68 @@ describe("scenarios > embedding > full app", () => {
       cy.button("Save this").should("not.exist");
     });
   });
+
+  describe("documents > comments", () => {
+    it("should not display comments in an embedded app", () => {
+      H.activateToken("bleeding-edge");
+      const DOCUMENT_ID = 1;
+      const PARAGRAPH_ID = "b7fa322a-964e-d668-8d30-c772ef4f0022";
+
+      H.createDocument({
+        idAlias: "documentId",
+        name: "Lorem ipsum",
+        document: {
+          type: "doc",
+          content: [
+            {
+              type: "paragraph",
+              attrs: {
+                _id: PARAGRAPH_ID,
+              },
+              content: [
+                {
+                  type: "text",
+                  text: "Lorem ipsum dolor sit amet.",
+                },
+              ],
+            },
+          ],
+        },
+      });
+      H.createComment({
+        target_type: "document",
+        target_id: DOCUMENT_ID,
+        child_target_id: PARAGRAPH_ID,
+        parent_comment_id: null,
+        content: {
+          type: "doc",
+          content: [
+            {
+              type: "paragraph",
+              attrs: { _id: uuid() },
+              content: [{ type: "text", text: "Test comment" }],
+            },
+          ],
+        },
+        html: "<p>Test comment</p>",
+      });
+
+      cy.intercept({
+        method: "GET",
+        path: "/api/ee/document/*",
+      }).as("documentGet");
+
+      H.visitFullAppEmbeddingUrl({
+        url: `/document/${DOCUMENT_ID}`,
+      });
+
+      cy.wait("@documentGet");
+
+      cy.findByLabelText("Show all comments").should("not.exist");
+
+      cy.findAllByRole("link", { name: "Comments" }).should("not.exist");
```

Comment:
I'd also add an intercept for fetching comments and assert that the endpoint was never called.